src/server/items/models.js

```diff
@@ -1,6 +1,11 @@
 import db from '../database';
 import {KeyUtils, STATE, CATEGORY, DEFAULT_PRECISON} from '../key-utils';
 
+export const STATE_STRING = {
```

Comment:
I thought this var have to be here cause it's not about 'key'.
Anyway i think so it's ambiguous. How you think which one is more proper?